### PR TITLE
chore(tasklist): clarify access privileges of tasklist user

### PR DESCRIPTION
**Pull request by MaxTru**
related to camunda/tasklist#2387

## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by vsgoulart — DISMISSED** [review]

File: docs/components/tasklist/userguide/using-tasklist.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ User tasks need an interaction from the user. This can be updating, adding varia
 
 If the user claimed a task, the task can be completed. Different task status filters help the user choose the desired task.
 
+Please note that when a user is granted tasklist access, the user will have full access to respective process instance data.
```
Comment (christinaausley) [inline docs/components/tasklist/userguide/using-tasklist.md:15]: I'd recommend noting this as an admonition. For example (with a few technical adjustments):
:::note
When a user is granted Tasklist access, the user has full access to the respective process instance data.
:::

Comment (MaxTru) [reply docs/components/tasklist/userguide/using-tasklist.md]: Done :+1:

**Review by christinaausley — APPROVED** [review]
Thanks for incorporating my suggestion 👍

**Comment by akeller** [comment]
From the technical writing perspective, this is good to merge, but based on the linked issue and the description of this issue, I wasn't sure if there was an ongoing conversation. @MaxTru please merge or apply a `hold` label.

**Comment by MaxTru** [comment]
> Thanks for incorporating my suggestion 👍

Thanks for the suggestion :-)

**Pull request merged**